.github/workflows/ci.yml

```diff
@@ -43,6 +43,11 @@ jobs:
     runs-on: sfdc-ubuntu-latest
     steps:
       - uses: "actions/checkout@v2"
+      - run: |
+          git fetch --no-tags origin main
+          git switch main
```

Comment:
This assumes the PR is against main, is it for backwards compatibility?

The `$GITHUB_BASE_REF` env var is available as an alternative. If you use the refspec form (`[...] origin $GITHUB_BASE_REF:$GITHUB_BASE_REF`) you can set up the local tracking branch during the fetch.

Reply:
Only works on certain kind of github action events, and "push" is not one of them. I think it's acceptable to just compare against main for now.

Reply:
Right. We switched to `pull_request` in SFDO-Tooling/CumulusCI@aa74f8d because of this. The problem is the noise: PRs against child branches will trigger spurious failures if main is updated with lines that aren't in the coverage report.

Other reasons to switch to `pull_request_target`:
1. External contributors don't push to the repo and [won't trigger workflows](https://github.com/SFDO-Tooling/Snowfakery/pull/727/commits/e8908e4a8af92cda72ba8d97444465354eafa161).
2. Build minutes cost money since the switch to premium runners.

That said, I'm fine with switching to a different event in a separate PR.